app/models/meetup.server.ts

```diff
@@ -0,0 +1,51 @@
+import type { MeetupGroupResponse, MeetupRequestBody } from "~/models/meetup";
+
+const REMIX_AUSTIN_MEETUP_URL = "remix-austin";
+const MEETUP_API = "https://www.meetup.com/gql";
+const JSON_HEADERS: HeadersInit = { Accept: "application/json" };
+
+// TODO use gql utils for typing
+const GET_GROUP_QUERY = `
+  query($urlname: String!) {
+   groupByUrlname(urlname: $urlname) {
+    link
+    memberships {
+     count
+    }
+    upcomingEvents(input: {first: 1}){
+     edges {
+      node {
+       title
+       shortUrl
+       venue {
+        name
+        address
+        city
+        state
+       }
+       dateTime
+       going
+      }
+     }
+    }
+   }
+  }
+`;
+
+async function getMeetupGroupInfo(url: string) {
+  const body: MeetupRequestBody = {
+    variables: { urlname: url },
+    query: GET_GROUP_QUERY,
+  };
+  return fetch(MEETUP_API, {
+    method: "POST",
+    body: JSON.stringify(body),
+    headers: JSON_HEADERS,
+  }).then((res) => res.json() as Promise<MeetupGroupResponse>);
+}
```

Comment:
We should probably parse to make sure this value is correct instead of type casting. Additionally, do we have any error handling?

Also, as I'm looking at this, if you want to use luxon maybe do it here in the server, that way we won't be sending the entire package just to massage a single field.

Definitely don't hesitate to message this data into the shape you want before sending it down the loader, no need to lots of work in the component when it can be done at the loader level.